Tests/MongoSwiftTests/ChangeStreamTests.swift

```diff
@@ -422,7 +422,7 @@ final class ChangeStreamTests: MongoSwiftTestCase {
             let stage = pipeline[0]
             let streamDoc = stage["$changeStream"] as? Document
             expect(streamDoc).toNot(beNil())
-            return streamDoc!.filter { $0.key != "startAtOperationTime" }
+            return streamDoc!.filter { $0.key != "resumeAfter" }
```

Comment:
this change is due to a bug in libmongoc's resume logic that was fixed in 1.15.1: https://github.com/mongodb/mongo-c-driver/commit/dc2f9c1ea27699727a7e2349f1310178e7630c09

Reply:
actually, I don't think this is the cause, will keep investigating

Reply:
ok, I see why this happened now. as of libmongoc 1.15.0, libmongoc will cache the `postBatchResumeToken` from the initial aggregate command and use that for resuming rather than the operation time, per the spec.
`postBatchResumeToken` exists starting on 4.0.7, and we only run this test on our tested versions that support `failCommand`, which as of now are all greater than 4.0.7.